site/tests/app/entities/poll/PollTester.php

```diff
@@ -35,13 +38,14 @@ public function setUp(): void {
                 "Poll #3",
                 "What is your favorite color?",
                 "multiple-response-survey",
+                new DateInterval("PT1M30S"),
                 new DateTime('now'),
                 "when_ended",
                 "when_ended",
                 "/var/local/submitty/courses/s21/sample/uploads/polls/poll_image_3_colors.png"
             )
         ];
-
+        $this->my_polls[0]->setClosed();
```

Comment:
Why is this change necessary?

Reply:
When a poll is constructed the endDate is set to the beginning of time '1900-02-01' if I don't set it closed, it won't correctly test the functionality. It will just compare with '1900-02-01' for each poll.

Reply:
Wait, you are right it shouldn't be necessary.

Reply:
Fixed